src/components/Loader/pages/CustomNavLink.tsx

```diff
@@ -0,0 +1,25 @@
+import classNames from 'classnames';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+interface Props {
+  to: string;
```

Comment:
Let's use `To` interface from `react-router-dom`
```suggestion
  to: To;
```